### ISSUE TITLE
After configuring for the first time, sensors/commands cannot be added before restarting the service

### ISSUE THREAD

**Issue by sleevezipper** [sleevezipper/hass-workstation-service#78]
A reboot also helps

**Comment by davewasthere** [comment]
Yeah, very weird behaviour prior to reboot (it appears as though no configuration values save, and can't add sensors. But reboot definitely helped.

Very minor, would be great to edit a sensor (e.g. change report frequency) without having to delete/re-add.
